index.html

```diff
@@ -2865,6 +2865,38 @@ <h2>Syntaxes</h2>
 CBOR, that is capable of expressing the data model.
       </p>
 
+      <p>
+One of the goals of this specification is to provide a flexible data model
+that can be expressed in a variety of syntaxes while also maximizing
+interoperability between the syntaxes. Implementers are advised to pay
+particular attention to the rules for serializing and deserializing data
+expressed in Sections <a href="#basic-concepts"></a> and
+<a href="#advanced-concepts"></a>. For example, while it may seem like
+serializing the <code>issuanceDate</code> value could be accomplished with a
+variety of date-time formats, the reality is that there is exactly one way to
+do so and the specification is explicit about this requirement.
+      </p>
+
+      <p>
+Syntaxes that provide intermediate expressions of the data model, one of which
+is detailed in Section <a href="#json-web-token"></a>, are allowed as long as
+the final syntactic expression that asserts conformance with this
+specification is truly conformant. Intermediate expressions are typically
```

Comment:
Saying something that asserts conformance needs to be "truly" conformant is odd/redunant. Let's simplify to: "are allowed as long as the final syntactic expression is conformant with this specification."